# std_msgs/Header header

float32 risk_X
float32 risk_Y
float32 risk_Z
float32 risk_SEP
string type_of_risk